src/test/java/uk/gov/pay/directdebit/payments/dao/PaymentViewDaoIT.java

```diff
@@ -372,14 +372,11 @@ public void shouldReturn1Records_whenSearchingByUserCancelNotEligibleExternalSta
                 .insert(testContext.getJdbi());
         aPaymentFixture()
                 .withMandateFixture(mandateFixture)
-                .withState(PaymentState.USER_CANCEL_NOT_ELIGIBLE)
+                .withState(PaymentState.SUBMITTED_TO_PROVIDER)
```

Comment:
Can this test go completely since its named `shouldReturn1Records_whenSearchingByUserCancelNotEligibleExternalState` and that state is no longer?

Reply:
fixed. good spot thanks!